internal/sdk/cloudian/sdk.go

```diff
@@ -236,6 +236,43 @@ func (client Client) CreateUser(ctx context.Context, user User) error {
 	return resp.Body.Close()
 }
 
+// CreateUserCredentials creates a new set of credentials for a user.
+func (client Client) CreateUserCredentials(ctx context.Context, user User) (*SecurityInfo, error) {
+	req, err := client.newRequest(ctx, client.baseURL+"/user/credentials", http.MethodPut, nil)
```

Comment:
should the second argument be simply `"/user/credentials"` as the baseURL now is internalized?

Reply:
Yeah, as soon as we merge https://github.com/statnett/provider-cloudian/pull/84 I'll update here